alchemy-web/docs/guides/cloudflare-nuxt-pipeline.md

```diff
@@ -136,6 +136,34 @@ declare module 'cloudflare:workers' {
 }
 ```
 
+## Create Worker Entry Point
+
+Create an `index.ts` file in the root of your project to handle requests:
+
+```typescript
+// index.ts
+/// <reference types="@cloudflare/workers-types" />
+import type { WorkerEnv } from "./env";
+// @ts-ignore - Suppress type errors if the module isn't found during editing/linting
+import nitroApp from "./.output/server/index.mjs";
+
+export default {
+  async fetch(
+    request: Request,
+    environment: WorkerEnv,
+    ctx: ExecutionContext
+  ): Promise<Response> {
+    const url = new URL(request.url);
+
+    if (url.pathname.startsWith("/api/")) {
+      return nitroApp.fetch(request, environment, ctx);
+    }
+
+    return environment.ASSETS.fetch(request);
```

Comment:
I don't think we need this conditional and asset fetch anymore. 

Cloudflare will serve the assets directly and our worker won't be called. You can just call nitroApp.fetch directly. 

Worth testing.